web/src/app/util/SkipToContentLink.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import Link from '@material-ui/core/Link'
+import { makeStyles } from '@material-ui/core'
+
+const useStyles = makeStyles({
+  skipLink: {
+    position: 'absolute',
+    height: '1px',
+    width: '1px',
+    overflow: 'hidden',
+    clip: 'rect(1px, 1px, 1px, 1px)',
+    '&:hover, &:focus, &:active': {
+      top: '5px',
+      left: '5px',
+      zIndex: 100000,
+      clip: 'auto !important',
+      display: 'block',
+      width: 'auto',
+      height: 'auto',
+      padding: '15px 23px 14px',
+      fontWeight: 'bold',
+      fontSize: '14px',
+      textDecoration: 'none',
+      lineHeight: 'normal',
+      color: '#CD1931',
+      backgroundColor: '#ffffff',
+      borderRadius: '3px',
+      boxShadow: '0 0 2px 2px rgba(0, 0, 0, 0.6)',
```

Comment:
It may be useful to use a MUI button here so that so much legwork doesn't need to be done for it to render properly. You should be able to accomplish the same functionality with a Link with a Button with `onClick="window.location.href = '#content';"`

Let me know what you think.

Reply:
I will try and find a way to make it look like an MUI button, but for accessibility/semantics, it should be an anchor tag.

Reply:
I changed it using the variant as Button, but I have reservations using it because it transforms text to uppercase which different screenreaders will interact with differently. Some will read out the text normally, whereas others will read each character individually. If you prefer to do this, I would want to override that property.

Reply:
I wonder if I could write a SkipContent component or variant to links for material-ui that is added to their repo🧐